src/app/pages/investment/investment.component.ts

```diff
@@ -0,0 +1,188 @@
+import { Component } from '@angular/core';
+
+import {
+  AUC_SORT_DIRECTION, AucDialogService,
+  AucTableHeaderItem,
+  AucTableRow, AucTransferModalComponent,
+  AucTransferModalData
+} from '@applicature/components';
+import { AS_COLOR_GROUP } from '@applicature/styles';
+
+const TableData: AucTableRow[] = [
```

Comment:
move it to a separate file. Create un a root folder directory mocks and place it here